### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by bryanhmiller**
Please know how much it kills me to turn in something that looks like this, but I believe it meets all the criteria.  @zoeames

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	<article>
+		<h2>Biographyish</h2>
+		<table>
+			<tr>
+    			<th>Name:</th>
+    			<td>Bryan</td>
+    			<td>Miller</td>
+  			</tr>
+  			<tr>
+    			<th>Born:</th>
```
Comment (zoeames) [inline index.html]: Don't put your age in here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    			<td>January</td>
+  			  	<td>1971</td>
+ 			</tr>
+  			<tr>
+    			<th>Place:</th>
+    			<td>Dallas</td>
+  			  	<td>Texas</td>
+ 			</tr>
+		</table>
+		<p>I was born in Texas, and grew up with the best BBQ in the world.  Totally irrelavant to the conversation at hand, but I seem to include it every time I speak of Texas.  When it comes to design, good design is always instantly recognizable, and so is bad design.  Working in the sign industry, we got to do some really cool things, and we made our share of bad ideas come to life, because the customer was paying for it.  It made me appreciate clean simple design all the more.</p>
```
Comment (zoeames) [inline index.html]: This is a good start but it says nothing about the fact that you are trying to become a software developer.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (zoeames) [inline index.html]: indent

**Comment by zoeames** [comment]
Hey a great start here!  I like the way you have added comments to your code.  Flush out your about me section a little.  Other than having links to dev related sites you never say that you are in school to become a jr developer.  Also check the indenting in your html file.  Head and body should be indented inside html

**Comment by bryanhmiller** [comment]
@zoeames I have addressed the issues that you mentioned.  I am ready for a second review.

**Comment by zoeames** [comment]
Very nice!  I am approving and merging in! 👍

**Pull request merged**